spec/flipper/adapters/mongo_spec.rb

```diff
@@ -16,8 +17,19 @@
 
   before do
     begin
-      collection.drop
-    rescue Mongo::Error::OperationFailure
+      Timeout::timeout(1) do
```

Comment:
Can we just set the timeout in the mongo client initialization and call it good? A rescue would be good, but I'd rather just use their timeout than a timeout like this.